package.json

```diff
@@ -49,7 +49,7 @@
 	],
 	"license": "MIT",
 	"dependencies": {
-		"eslint": "^7.6.0",
+		"eslint": "^7.8.0",
```

Comment:
Do you want to actually pin to 7.8.1+? This is 7.8.0+.

Reply:
Yes, apparently I only pushed package-lock.json